src/pages/patientView/PatientViewPage.tsx

```diff
@@ -200,6 +201,18 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
                     sampleManager.clinicalDataLegacyCleanAndDerived[sample.id] &&
                     sampleManager.clinicalDataLegacyCleanAndDerived[sample.id].DERIVED_NORMALIZED_CASE_TYPE === 'Xenograft'
                 );
+                // check whether the sample has a Genomic Driven Diagnosis
+                // prediction
+                const hasGDDPred:boolean = (
+                    AppConfig.genomeDrivenDiagnosisUrl &&
+                    AppConfig.genomeDrivenDiagnosisUrl.length > 0 &&
+                    patientViewPageStore.genomeDrivenDiagnosis.isComplete &&
+                    patientViewPageStore.genomeDrivenDiagnosis.result &&
+                    patientViewPageStore.genomeDrivenDiagnosis.result[sample.id] &&
+                    _.every(["Pred1","Pred2","Pred3","Conf1","Conf2","Conf3"]
```

Comment:
this is a bit of a strange use of _.every i think, thought it works.   i think you don't need the map.  just pass as second param to every the callback you are passing to map

Reply:
also, this should probably go into a computed getter on this component.

Reply:
yes good points. Im just gonna add this to an issue that the sampleHeader should be it's own component. We prolly change this a bit more once it goes through beta testing. Also the data might come from clinical attributes instead once we get it fed from cvr. So not super keen on restructuring this rn: https://github.com/cBioPortal/cbioportal-frontend/issues/988